src/main/scala/performanceanalysis/logreceiver/alert/AlertActionActor.scala

```diff
@@ -0,0 +1,14 @@
+package performanceanalysis.logreceiver.alert
+
+import akka.actor.{Actor, ActorLogging, Props}
+
+object AlertActionActor {
+
+  def props(): Props = Props.apply(new AlertActionActor)
+}
+
+class AlertActionActor extends Actor with ActorLogging {
+
+  override def receive: Receive = ???
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Usage of ??? operator](https://www.codacy.com/app/scala-academy/performance-analysis/file/2514469703/issues/source?bid=3220915&fileBranchId=3273607#l12)